apps/omg_child_chain/lib/omg_child_chain/datadog_event/contract_event_consumer.ex

```diff
@@ -34,10 +34,10 @@ defmodule OMG.ChildChain.DatadogEvent.ContractEventConsumer do
   # sobelow_skip ["DOS.StringToAtom"]
   @spec prepare_child(keyword()) :: %{id: atom(), start: tuple()}
   def prepare_child(opts) do
-    %OMG.Bus.Topic{topic: topic} = Keyword.fetch!(opts, :topic)
+    topic = Keyword.fetch!(opts, :topic)
```

Comment:
Is this our tuple? I guess so, then we could
```suggestion
    {origin, topic} = Keyword.fetch!(opts, :topic)
```

and then 👇